app/repositories/DepartureRepository.scala

```diff
@@ -518,6 +522,30 @@ class DepartureRepository @Inject()(
         }
     }
   }
+
+  def deleteCompletedJourneys(): Future[Try[Unit]] = {
+    val writeOffSelector = Json.obj("messages.messageType" -> WriteOffNotification.code)
+    val rejectedSelector = Json.obj("messages.messageType" -> DeclarationRejected.code)
+    val messageSelector  = Json.obj("$or"                  -> Json.arr(writeOffSelector, rejectedSelector))
```

Comment:
Kind of nitpicky but you could do this instead
```suggestion
    val messageSelector  = Json.obj(
      "messages.messageType" -> Json.obj(
        "$in" -> Json.arr(WriteOffNotification.code, DeclarationRejected.code)
      )
    )
```

Reply:
I wonder if that changes the logic that's run on the server?